src/icon.c

```diff
@@ -351,7 +357,7 @@ GdkPixbuf *icon_get_for_data(GVariant *data, char **id, double dpi_scale, int mi
                                           width,
                                           height,
                                           rowstride,
-                                          (GdkPixbufDestroyNotify) g_free,
```

Comment:
@fwsmit I replaced this g_free with an actual function taking two arguments since it just seemed wrong. Was this supposed to free both the pixels and data_pb?